packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -7,29 +7,1024 @@ category: management
+#### Request body
+
+This request requires the `verificationCode` property as its request body.
+
+| Property | Type                     | Description                          |
+| -------- | -------------------------|--------------------------------------|
+| `verificationCode`  | String | A 6-digit verification code that the user receives via `SMS` or `CALL` |
+
+#### Response body
+
+Returns an empty response with an HTTP 204 status code.
+
+Disabling the factor type of the corresponding method on the org returns a 403 FORBIDDEN with error code E0000038.
+
+Disabling the `IDP_MY_ACCOUNT_API` feature flag on the org returns a 401 UNAUTHORIZED with error code E0000015.
+
+Passing an invalid `id` returns a 404 NOT FOUND with error code E0000008.
+
+Passing an invalid `verificationCode` returns a 400 BAD REQUEST with error code E0000001.
+
+Passing a valid `verificationCode` again (verifying a phone number that has been verified) results a no-op.
+
+In an extremely rare case, verifying a phone number when the challenge is about to expire returns a 409 CONFLICT with error code E0000157.
+
+Failing to answer the challenge (possibly due to invalid `verificationCode`) returns a 401 UNAUTHORIZED with error code E0000004.
+
+#### Usage example
+
+##### Request
+
+The following request verifies the phone number represented by `id` with a `verificationCode` of 796672.
+
+```bash
+curl -XPOST 'https://${yourOktaDomain}/myaccount/phones/{id}/verify' -H 'Authorization: bearer {token}' -H 'Content-Type: application/json' --data '{
+     "verificationCode": "796672"
+ }'
+```
+
+### Delete My Phone
+
+<ApiOperation method="delete" url="/idp/myaccount/phones/{id}" />
+
+Delete the current user's phone information by id.
+
+#### Required scope and role
+
+An Okta scope of `okta.myAccount.phone.manage` is required to use this endpoint.
+
+> **Note:** Admin users are not allowed to call the `/idp/myaccount/phones/{id}/` DELETE endpoint.
+#### Request path parameters
+| Parameter  | Type   | Description                                       |
+| ---------- | ------ | ------------------------------------------------- |
+| `id` | String | ID of the phone. The ID of the phone can be obtained through `GET /idp/myaccount/phones` or `POST /idp/myaccount/phones` when adding a new phone. |
+
+#### Request query parameters
+
+N/A
+
+#### Response body
+
+N/A
+
+#### Usage example
+
+Any non-admin user with a valid bearer token can issue this request to delete their phone.
+
+##### Request
+
+```bash
+curl -v -X DELETE \
+-H "Authorization: Bearer ${token}" \
+"https://${yourOktaDomain}/idp/myaccount/phones/{id}"
+```
+
+##### Response
+
+Returns an empty HTTP 204 status code response.
+
+#### Error Responses
+
+If the phone authenticator is not enabled for `method` on the org, the response will return a 403 FORBIDDEN with error code E0000038.
+
+If an invalid phone id is passed to the request, the response will return a 404 NOT FOUND with error code E0000008.
 
 ### Get Me
```

Comment:
if we have another doc for the old profile endpoint we should remove this endpoint and all the reference all toghether.

Reply:
@JayHe-okta Yes, we are keeping the existing MyAccount API docs in a hidden spot. Only accessible by link from the new MyAccount API docs.

So you want me to remove the following:
- lines 1022-1070
- lines 1539-1570

Thanks!

Reply:
thanks Brent, yes the line number is correct lets remove them, the get me endpoint is not part of the new myaccount api cc: @justinabrokwah-okta

Reply:
Done!